.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
     branches:
       - main
+      - version-1.0
```

Comment:
Remove? Or make a note for later. You can make these PRs against main?

Reply:
I'll add a note.  But I want to have a version-1.0 branch to work against while we work on the features for that release.  I'm not sure the timeline for completing and releasing so don't want to use main.